README.md

```diff
@@ -881,6 +883,31 @@ ESLint rules (applies this guide as linting rules):
 
   const luke = new Jedi();
 
+  luke
+    .jump()
+    .setHeight(20);
+    
+  // better
```

Comment:
just making sure - is there no 'best' for this rule?

Reply:
there are two good options ... `better` is grammatically correct, `best` is only applicable when there are three or more options.